### DISCUSSION TITLE
Open subtasks' view from story view

### DISCUSSION THREAD

**Discussion by harkairt** [ankitpokhrel/jira-cli#448] [Ideas]
My workflow would be:
- list the issues
- open a story with `v`
- navigate to a sub-task from within the "inner" `jira issue view`

**Comment by kandersen** [comment]
May I add: Starting from a sub-task on the issue list, be able to navigate _up_ the task hierarchy.

**Comment by Dmitrev** [comment]
Navigating to linked issues would also be useful
